### PR TITLE
Edit cart item

### LINKED ISSUE #98: Add editing of cart
In cart checkout page, there is a non-functional edit button

To implement it and allow users to edit quantity in cart checkout

### PR DESCRIPTION
**Pull request by jonasgwt**
# Description

* Fix styling issues in cart checkout page
* Add pop up modal for users to edit quantity in cart checkout
* Add tests for new pop up modal

Fixes #98 

## Type of change

- [ ] Bug fix 
- [x] New feature 
- [ ] Breaking change 

# How Has This Been Tested?

For withdrawing items:

https://github.com/sjb-deck/deck/assets/55698556/24539b87-9cfa-48a5-938a-e42d55337b50

For depositing items:

https://github.com/sjb-deck/deck/assets/55698556/f9b9d1a9-ff7b-4698-ac83-6adab2af48f3

### CODE REVIEW

File: __tests__/components/CartContent/EditPopUp/EditCartItemPopUp.test.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        cartItem={mockCartItem}
+        open={true}
+        handleClose={jest.fn()}
+      />,
+    );
+
+    expect(screen.getByText(mockCartItem.name)).toBeInTheDocument();
+    expect(screen.getByLabelText('Expiry Date')).toHaveValue(
+      mockCartItem.expiry_dates[0].expiry_date ?? 'No Expiry',
+    );
```
Comment (sweic) [inline __tests__/components/CartContent/EditPopUp/EditCartItemPopUp.test.js:34]: Add a test case when the expiry_date property is non-null, and should be displayed

**Review by sweic — APPROVED** [review]
